src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -14,6 +14,8 @@
     <PackageReference Include="DurableTask.AzureStorage" Version="0.1.0-alpha" />
     <PackageReference Include="Microsoft.AspNet.WebApi.Core" Version="5.2.3" />
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="2.1.0-beta1-10950" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Logging.ApplicationInsights" Version="2.1.0-beta1" />
```

Comment:
Can you update this package to the nightly version from myget? It looks like we're on 2.1.0-beta1-11013 right now.